api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerTls.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+/**
+ * Represent the TLS configuration for Kafka Mirror Maker
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class KafkaMirrorMakerTls implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private List<CertSecretSource> trustedCertificates;
+    private Map<String, Object> additionalProperties;
+
+    @Description("Trusted certificates for TLS connection")
```

Comment:
So these are the certificates the mirror maker needs to trust to connect to source and/or destination, right? It's worth explaining that this in more detail, I think.

Reply:
You still need to add more detail of precisely which Secrets containing which certs should be added here.

Reply:
wdym @ppatierno In other CRDs? Do we need them? In other cases perhaps not if they can be reasonably defaulted for the "current" cluster. For mirror maker, which is cross-cluster by definition I don't think such a default really helps matters very much because you'd always have to provide the Secrets for the other cluster. Question: If we're using separate stores how do to know which crts belong in which store?

Reply:
With the current CRD proposal (you can find the global proposal in this PR): 

```
consumer:
    tls:
     trustedCertificates:
      # as it works for Kafka Connect for example
producer:
   tls:
      trustedCertificates:
        # as it works for Kafka Connect for example
```

We have two distinct sections for consumer and producer configuration. 
The user has to provide two secrets (or the same secret) containing the CA certificate used to sign brokers certificates for the source and target cluster. It's the same way how Kafka Connect works for having the freedom to connect to any Kafka cluster (and not ones deployed by Strimzi only). Maybe I miss understood your point.

Reply:
OK, my bad for not looking at how this class is used.